crates/re_viewport/src/viewport_blueprint_ui.rs

```diff
@@ -68,6 +66,70 @@ impl Viewport<'_, '_> {
         };
     }
 
+    /// Display the root container.
+    ///
+    /// The root container is different from other containers in that it cannot be removed or dragged, and it cannot be
+    /// collapsed, so it's drawn without a collapsing triangle..
+    fn root_container_tree_ui(&self, ctx: &ViewerContext<'_>, ui: &mut egui::Ui) {
+        let Some(container_id) = self.blueprint.root_container else {
+            // nothing to draw if there is no root container
+            return;
+        };
+
+        let Some(container_blueprint) = self.blueprint.containers.get(&container_id) else {
+            re_log::warn_once!("Cannot find root container {container_id}");
+            return;
+        };
+
+        let mut visibility_changed = false;
+        let mut visible = container_blueprint.visible;
+        let parent_visible = true;
+        let container_visible = visible;
+        let item = Item::Container(container_id);
+
+        let item_response = ListItem::new(
+            ctx.re_ui,
+            format!("Viewport ({:?})", container_blueprint.container_kind),
+        )
+        .subdued(!container_visible)
+        .selected(ctx.selection().contains_item(&item))
+        .draggable(false)
+        .drop_target_style(self.state.is_candidate_drop_parent_container(&container_id))
+        .label_style(re_ui::LabelStyle::Unnamed)
+        .with_icon(crate::icon_for_container_kind(
+            &container_blueprint.container_kind,
+        ))
+        .with_buttons(|re_ui, ui| {
+            let response = visibility_button_ui(re_ui, ui, parent_visible, &mut visible);
+            visibility_changed = response.changed();
+            response
+        })
```

Comment:
what's the usecase of a visibility button on the root container? Just for consistency? I'd suggest dropping it

Reply:
That.... is a good point. I'll admit not having thought that through 🙈  Who cares about blanking the viewport?